hack/build-antrea-windows-all.sh

```diff
@@ -69,7 +69,8 @@ pushd "$THIS_DIR/.." > /dev/null
 NANOSERVER_VERSION=$(head -n 1 build/images/deps/nanoserver-version)
 CNI_BINARIES_VERSION=$(head -n 1 build/images/deps/cni-binaries-version)
 GO_VERSION=$(head -n 1 build/images/deps/go-version)
-WIN_BUILD_TAG=$(echo $GO_VERSION $CNI_BINARIES_VERSION $NANOSERVER_VERSION| md5sum| head -c 10)
+OVS_VERSION_WINDOWS=$(head -n 1 build/images/deps/ovs-version-windows)
+WIN_BUILD_TAG=$(echo $GO_VERSION $CNI_BINARIES_VERSION $NANOSERVER_VERSION $OVS_VERSION_WINDOWS| md5sum| head -c 10)
```

Comment:
WIN_BUILD_ARGS  in Makefile should also be updated.

Reply:
Also, I think it's better to name it WIN_OVS_VERSION to align with CNI_BINARIES_VERSION and NANOSERVER_VERSION.